content/webapp/views/pages/concepts/concept/index.tsx

```diff
@@ -331,7 +330,7 @@ const ConceptPage: NextPage<Props> = ({
     if (relatedTopics?.length && personOrAllFields) {
       links.push({
         text: relatedConceptsGroupLabel,
-        url: `#${dasherize(relatedConceptsGroupLabel)}`,
+        url: `#related-topics`,
```

Comment:
We want to keep the url hash consistent for tracking (maybe this needs commenting)